runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -386,23 +380,6 @@ public void stop() {
         log.debug("stop: Shutting down router for {}", node);
         shutdown = true;
         connectionFuture.completeExceptionally(new ShutdownException());
-        try {
-            channel.disconnect();
-            channel.close().syncUninterruptibly();
```

Comment:
Can we keep these two channel closing lines but modify the second one:
```
channel.disconnect();
channel.close();
```
I think we should still request the channel to close, but just don't wait on it.

Reply:
Closing the event loops will call all child channels. 
Look here:
https://stackoverflow.com/questions/37902844/netty-serversocket-and-children-graceful-shutdown

Reply:
Sorry I may not get your point, since we remove the codes to shutdown the event loops, how can they call all child channels?

Reply:
shutting down the event loop will shut down the channels.
https://netty.io/wiki/user-guide-for-4.x.html#shutting-down-your-application

Reply:
no its still there:
https://github.com/CorfuDB/CorfuDB/pull/1577/files#diff-ec92d60e991399ec1c89e259082b7965R693

Reply:
I see, thanks for clarification.